code/modules/spells/spell.dm

```diff
@@ -446,7 +446,7 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell)) //needed for th
 					chosen_target = tgui_input_list(user, "Choose the target for the spell", "Targeting", sort_names(possible_targets))
 					if(isnull(chosen_target))
 						return
-					if(ismob(chosen_target) || user.incapacitated())
+					if(user.incapacitated())
```

Comment:
```suggestion
					if(!ismob(chosen_target) || user.incapacitated())
```

Reply:
![image](https://user-images.githubusercontent.com/53777086/149303076-3d9b796d-2c59-4b87-a323-928ea0a6a752.png)

Reply:
the possible targets are hardcoded to be a only `mob/living`s in view few lines up, so if someone wanted a targeting spell for non-mobs they'd need more adjustments anyways